src/sql/workbench/contrib/connection/browser/connectionStatus.ts

```diff
@@ -59,16 +64,38 @@ export class ConnectionStatusbarItem extends Disposable implements IWorkbenchCon
 	// Update the connection status shown in the bar
 	private _updateStatus(): void {
 		let activeConnection = TaskUtilities.getCurrentGlobalConnection(this._objectExplorerService, this._connectionManagementService, this._editorService, this._logService);
+		let id = undefined;
+		if (this._editorService.activeEditor) {
+			// USE ACTIVE EDITOR INFO AS THE ID WILL BE DIFFERENT FOR EDITOR CONNECTION.
```

Comment:
nit: Would suggest not to use caps. I would also suggest explaining a bit more what this means - it's not clear from the comment why the connection would be different in this case